test/unit/test_inference.py

```diff
@@ -29,7 +29,6 @@
 _EOS = -1
 
 
-@pytest.fixture
```

Comment:
wait, shouldn't it be enough to remove this fixture decorator? The warning was about calling the fixture directly, not about having fixtures in general.